### DISCUSSION TITLE
Lost sorting option in v2.7.1

### DISCUSSION THREAD

**Discussion by senthil13** [goharbor/harbor#18500] [General]
Hi - We were having the sorting option (particularly in the project quota screen), but now after the upgrade, we are not able to see that. Did it removed in the latest version?

**Comment by zyyw** [comment]
Hi there, which sorting option you are referring to? could you please add a screenshot to show it? thanks

**Comment by AllForNothing** [comment]
@senthil13 We refactored the APIs in v2.0.0, and some sorting options had been removed. For project quota, you can refer to this https://github.com/goharbor/harbor/issues/16517. We are planning to deliver this in v2.9

**Reply by senthil13** [reply]
Thanks for the response.
